testsuite/openshift/objects/auth_config.py

```diff
@@ -23,7 +23,8 @@ class AuthConfig(OpenShiftObject, Authorization):
     """Represents AuthConfig CR from Authorino"""
 
     @classmethod
-    def create_instance(cls, openshift: OpenShiftClient, name, host, labels: Dict[str, str] = None):
+    def create_instance(cls, openshift: OpenShiftClient, name, host, labels: Dict[str, str] = None,
+                        response: Dict = None):
```

Comment:
This should be added as method to Authorization (add_response) instead of init parameter